content/docs/connectors/oauth.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Authentication Through an Generic OAuth 2.0 Provider"
+linkTitle: "Generic OAuth 2.0"
+description: ""
+date: 2021-03-15
+draft: false
+toc: true
+weight: 2050
+---
+
+## Overview
+
+Dex users can make use of this connector to work with standards-compliant [OAuth 2.0](https://oauth.net/2/) authorization provider, in case of that authorization provider is not in the Dex connectors list.
+
+## Configuration
+
+The following is an example of a configuration for using OAuth connector with Reddit.
+
+```yaml
+connectors:
+- type: oauth
+  # ID of OAuth 2.0 provider
+  id: reddit 
+  # Name of OAuth 2.0 provider
+  name: reddit
+  config:
+    # Connector config values starting with a "$" will read from the environment.
+    clientID: $REDDIT_CLIENT_ID
+    clientSecret: $REDDIT_CLIENT_SECRET
+    redirectURI: http://127.0.0.1:5556/callback
+
+    tokenURL: https://www.reddit.com/api/v1/access_token
+    authorizationURL: https://www.reddit.com/api/v1/authorize
+    userInfoURL: https: https://www.reddit.com/api/v1/me
+ 
+    # Optional: Specify whether to communicate to Auth provider without validating SSL certificates
+    # insecureSkipVerify: false
+
+    # Optional: The location of file containing SSL certificates to commmunicate to Auth provider 
```

Comment:
I suggest breaking this line so we don't render the horizontal scrollbar in the code snippet box
```suggestion
    # Optional: The location of file containing SSL certificates to commmunicate
    # to Auth provider 
```